src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -450,6 +456,50 @@ public function newInstanceArgs(array|null $args = null): object
         throw new Exception\NotImplemented('Not implemented');
     }
 
+    /** @param int-mask-of<self::SKIP_*> $options */
+    public function newLazyGhost(callable $initializer, int $options = 0): object
+    {
+        throw new Exception\NotImplemented('Not implemented');
```

Comment:
I think we should throw `Exception\NotImplementedByDesign()` or such, or perhaps `Exception\NotImplementedBecauseItTriggersAutoloading()` or such.

This would signal (to other people using the library) that this will never work in BetterReflection.

I wonder if we should also do an `@return never` on each of these

Reply:
Added new exception and `@return never` in last two commits.